test/unit/helpers/admin/cancel_routes_helper_test.rb

```diff
@@ -0,0 +1,24 @@
+require "test_helper"
+
+class Admin::CancelRoutesHelperTest < ActionView::TestCase
+  test "admin_cancel_path take a new edition instance and generates the correct path" do
+    e = build(:edition)
+    assert_equal "/government/admin/editions", admin_cancel_path(e)
+  end
+
+  test "admin_cancel_path take a new corporate_information_page instance and generates the correct path" do
+    c = build(:corporate_information_page)
+    c.organisation = create(:organisation)
```

Comment:
Looking great. This is a bit nitty, but i generally go with `build_stubbed` rather than `create` wherever possible in unit tests as it's quicker

There's a little article on it here https://kolosek.com/factory-girl-build-vs-build_stub/

`create` - instantiates the object and associations
`build` - builds the object and instantiates associations 
`build_stubbed` - creates and assigns attributes to an object to make it behave like an instantiated object, but does not hit the database